app/src/main/java/com/vanniktech/emoji/sample/EmojiApplication.java

```diff
@@ -10,6 +10,10 @@
   @Override public void onCreate() {
     super.onCreate();
 
+    if (LeakCanary.isInAnalyzerProcess(this)) {
+      return;
```

Comment:
No, have a look at the [sample](https://github.com/square/leakcanary/blob/master/leakcanary-sample/src/main/java/com/example/leakcanary/ExampleApplication.java#L29) and the [FAQ](https://github.com/square/leakcanary/wiki/FAQ#how-do-i-use-it). It is also explained what this line is for there.